spec/Ranges.tex

```diff
@@ -475,8 +475,10 @@ \subsection{Range Assignment}
 \item An implicit conversion is allowed from \chpl{idxType} of the source range
        to \chpl{idxType} of the destination range type,
 \item the two range types have the same \chpl{boundedType}, and
-\item either the destination range is stridable or the source range's
-      stride is 1.
+\item either the destination range is stridable or the source range
+      is not stridable.
+\item Or an explicit conversion is used to make the source range match the
+      type of the destination range.
```

Comment:
I suggest dropping the last bullet in the above. If the source is an explicit conversion expression like (someRange : someType), then "someRange" is no longer the source of the assignment. That will take care of the and/or structure of the list. Which [i.e. restructuring], IMO, needs to be done one way or another, if we are to keep all four bullets.